parsely/src/main/java/com/parsely/parselyandroid/LocalStorageRepository.kt

```diff
@@ -70,10 +82,9 @@ internal open class LocalStorageRepository(private val context: Context) {
     /**
      * Save the event queue to persistent storage.
      */
-    @Synchronized
-    open fun persistQueue(inMemoryQueue: List<Map<String, Any?>?>) {
-        ParselyTracker.PLog("Persisting event queue")
-        persistObject((inMemoryQueue + getStoredQueue()).distinct())
+    open suspend fun insertEvents(toInsert: List<Map<String, Any?>?>) = mutex.withLock {
+        ParselyTracker.PLog("Persisting ${toInsert.size} events")
```

Comment:
Suggestion (💡): Just like it is done on `InMemoryBuffer.add(...)`, maybe this log can be moved to `InMemoryBuffer.init { ... }` as well and make the logging closer to source and more consistent, at least with each other. Wdyt? 🤔